data/blacklist.d/porn_custom.hosts

```diff
@@ -25,8 +25,48 @@
 127.0.0.1 vlxx.cc
 127.0.0.1 vlxx.com
 127.0.0.1 vlxx.tv
-127.0.0.1 vov.vn
 127.0.0.1 webphimsex.net
 127.0.0.1 www.sexmoi.net
 127.0.0.1 xnxx.com
 127.0.0.1 xvideos.com
+127.0.0.1 gaigoi18.com
+127.0.0.1 gai.to
+127.0.0.1 soc3x.com
+127.0.0.1 phim.sex
+127.0.0.1 phimsex.club
+127.0.0.1 ditvl.net
+127.0.0.1 phimsex1.net
+127.0.0.1 hdxnxx.mobi
+127.0.0.1 heo3x.tv
+127.0.0.1 javnong.com
+127.0.0.1 phimsexsub.tv
+127.0.0.1 javhd.pro
+127.0.0.1 phe3x.net
+127.0.0.1 londep.pro
+127.0.0.1 sex5.pro
+127.0.0.1 javphim.net
+127.0.0.1 javforme.ninja
+127.0.0.1 kichduc.cc
+127.0.0.1 zphimsex.com
+127.0.0.1 sexhayvl.pro
+127.0.0.1 sexngon.net
+127.0.0.1 phimmoi.net
+127.0.0.1 phimbathu.org
+127.0.0.1 fimfast.com
+127.0.0.1 fimfast.net
+127.0.0.1 melonstube.cc
+127.0.0.1 xxnx.fun
+127.0.0.1 vlphimsex.net
+127.0.0.1 javlibrary.tv
+127.0.0.1 cu3x.net
+127.0.0.1 jav321.net
+127.0.0.1 jav21.net
+127.0.0.1 top3x.net
+127.0.0.1 clipviet.net
+127.0.0.1 phimsexvn.tv
+127.0.0.1 phimsec.pro
+127.0.0.1 kphimsex.net
+127.0.0.1 phimsex.to
+127.0.0.1 tphimsex.net
+127.0.0.1 vidz7.club
+127.0.0.1 dphimsex.net
```

Comment:
Out of habit, I like to keep this file sorted a-z. But I'll accept this for now, since it is ready to go.